extensions/rvadapter/src/main/java/com/google/android/agera/rvadapter/RepositoryAdapter.java

```diff
@@ -128,7 +133,84 @@ public RepositoryAdapter build() {
       return new RepositoryAdapter(this);
     }
 
+    /**
+     * Builds the {@link RepositoryAdapter} that presents the provided repositories in order and
+     * observes the repositories as well as any additional observables while the provided
+     * {@link Activity} is active (between onResumed/onPaused).
+     */
+    @TargetApi(14)
+    @NonNull
+    public Adapter<ViewHolder> whileActive(@NonNull final Activity activity) {
```

Comment:
The state between onResume (no 'd') and onPause (no 'd') is technically called "resumed", not "active".

The other method can also be similarly renamed to 'whileStarted'.